src/models/properties/reducers.ts

```diff
@@ -13,7 +13,7 @@ import {
 import { getAsyncActionsReducer } from '../helpers/asyncActions';
 
 const statuses = Object.keys(PostStatusForApi)
-  .slice(0, 7)
```

Comment:
why do we need `.slice` here?

Reply:
Because in numeric enum , for example
```typescript
enum NumericEnum {
  One = 1,
  Two = 2,
  Three = 3,
  Four = 4,
}
```

Simply using the "Object.values()" or "Object.keys()" method will return both the keys and values in a combined array.

```javascript
const values = Object.values(NumericEnum);
console.log(values);  //👈  ['One', 'Two', 'Three', 'Four', '0', '1', '2', '3']

const keys = Object.keys(NumericEnum);
console.log(keys); //👈  ['0', '1', '2', '3', 'One', 'Two', 'Three', 'Four']
```

In order to get first six ids we use ```.slice(0,6)```

As an alternitive we can use ```filter``` method, for example: 

```javascript
const statuses = Object.keys(PostStatusForApi)
  .filter(value => !Number.isNaN(Number(value)))
  
  console.log(statuses); // 👈 ['0', '1', '3', '4', '5', '6']
```

Reply:
omg, I forgot about such a thing 🤦 
thank you for explanation

ps To me it's a good reason to avoid enums 😅